apps/api/src/app/workflows-v2/usecases/sync-to-environment/sync-to-environment.usecase.ts

```diff
@@ -68,6 +76,33 @@ export class SyncToEnvironmentUseCase {
         session: command.session,
       })
     );
+
+    await this.publishTranslationGroup(sourceWorkflow.workflowId, command);
```

Comment:
@scopsy  Not sure if we want to do it here and right away?

Reply:
I think it's ok here, I will need to pass the session to be the same, but i will handle it. 

I wonder is this properly handlign when there workflow is not translated, or translation was turned of ?